try {
    la 
} catch(e) {
    console.log(e)
}